app/controllers/admin/domains/registry_lock_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Admin
+  module Domains
+    class RegistryLockController < BaseController
+      def destroy
+        set_domain
```

Comment:
Is it a better way than using `before_action` callback or `.helper_method`?

Reply:
Yes. there's exactly one method in this controller. `before_action` makes sense when you re-use something more than once.